### PR TITLE
vSphere: skip delete of VMs w/ova

### PR DESCRIPTION
**Pull request by jcpowermac**
Add a `grep -v ova` for UPI jobs so the template is not delete when it
remains on the ci-segment-#.

### CODE REVIEW

**Comment by jcpowermac** [comment]
ok...that broke something
/hold

File: ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh [hunk 1]
```diff
@@ -32,6 +32,14 @@ echo "$(date -u --rfc-3339=seconds) - Find virtual machines attached to ${LEASED
...
-# 4. Power off and delete the virtual machine
-govc ls -json -t OpaqueNetwork "/SDDC-Datacenter/network/${LEASED_RESOURCE}" | jq '.elements[]?.Object.Vm[]?.Value' | xargs -I {} govc ls -L VirtualMachine:{} | xargs -I {} govc vm.destroy {}
+# 4. skip the templates with ova
+# 5. Power off and delete the virtual machine
+
+govc ls -json -t OpaqueNetwork "/SDDC-Datacenter/network/${LEASED_RESOURCE}" |\
+    jq '.elements[]?.Object.Vm[]?.Value' |\
+    xargs -I {} govc ls -json -L VirtualMachine:{} |\
+    jq '(.elements[] | select(.Path | contains("ova") | not)).Path' |\
+    xargs -I {} govc vm.destroy {}
```
Comment (staebler) [inline ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh:42]: Do these xargs calls need `--no-run-if-empty` to cover the case where there is a network without any non-ova VMs attached?

**Comment by jcpowermac** [comment]
The rehearsal failures are not related to this change but just "normal" openshift-test failure.

**Pull request merged**